revolv/templates/project/project.html

```diff
@@ -13,223 +14,344 @@
+        <!-- end .banner -->
+        <div class="banner-section">
+            <div class="container">
+                <!--   <div class="title-white-border title-active-project pull-right">{{ project.status_display|upper }}</div><h1 class="title-h1">
+            {{ project.title }}
+        </h1> -->
+                <div class="funded-round">
+                    <span class="status-text">
+                        <span class="round-depict">
+                            <span>OF</span> ${{ project.funding_goal|floatformat:0|intcomma }}
+                        </span>
+                    </span>
+                    <!-- end .status-text -->
+                    <div class="status-indicator desktop-circle">
+                        <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="210" data-height="210" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".12">
+                        </div>
+                        <div class="status-indicator small-circle">
+                            <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="175" data-height="175" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".13">
+                            </div>
+                            <div class="status-indicator tablet-circle">
+                                <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="152" data-height="152" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".16">
+                                </div>
+                                <div class="status-indicator mobile-circle">
+                                    <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="75" data-height="75" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".12">
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- end .banner-section -->
+                        <div class="info-section">
+                            <div class="container">
+                                <div class="video-main">
+                                    <div class="row">
+                                    <div class="video-area col-lg-8 col-md-7">
+                                        <div class="video">
               {% if project.video_url %}
-                <iframe class="project_video youtube-player" allowfullscreen></iframe>
+                
+                                            <iframe class="project_video youtube-player" allowfullscreen></iframe>
```

Comment:
Please indent this line to be 4 spaces more than the `{% endif %}` on the following line.